bin/cmds/list.ts

```diff
@@ -10,8 +10,32 @@ export const describe = "List all projects and their actions";
 export async function handler(argv: any) {
 	try {
 		const config = await loadConfig(argv.cwd as string, argv.needs, false);
+		const action_obj: Record<string, Record<string, string[]>> = {};
+
 		for (const [name, project] of Object.entries(config.projects)) {
-			console.log(chalk`{bold ${name}:} {cyan [${Object.keys(project.actions).join(", ")}]}`);
+			const groups = Object.keys(project.actions);
+
+			// instantiate
+			action_obj[name] = {};
+
+			groups.forEach(function (group) {
+				action_obj[name][group] = Object.keys(project.actions[group].groups);
+			});
+		}
+
+		for (const name in action_obj) {
+			let group_action = "";
+			let i = Object.keys(action_obj[name]).length;
+
+			for (const group in action_obj[name]) {
+				if (i <= 1) {
+					group_action += chalk`{cyan <${group}>} {magenta <${action_obj[name][group]}>}`;
+				} else {
+					group_action += chalk`{cyan <${group}>} {magenta <${action_obj[name][group]}>}, `;
+				}
+				--i;
+			}
+			console.log(chalk`{bold ${name}:} ${group_action}`);
```

Comment:
I think the code structure would be more clear if we kept the previous log statement, but instead mapped the project.actions dictionary into a formatted string array

Such that the previous line 14 would look something like this
```Typescript
console.log(chalk`{bold ${name}:} {cyan [${Object.entries(project.actions).map(actionToPrettyString)}]}`);
```

And then define the function somewhere fitting, maybe in this file like `function actionToPrettyString(actionName: string, action: ProjectAction): string`